src/Controls/src/Core/RowDefinition.cs

```diff
@@ -6,13 +6,17 @@ namespace Microsoft.Maui.Controls
 	public sealed class RowDefinition : BindableObject, IDefinition, IGridRowDefinition
 	{
 		/// <include file="../../docs/Microsoft.Maui.Controls/RowDefinition.xml" path="//Member[@MemberName='HeightProperty']/Docs" />
-		public static readonly BindableProperty HeightProperty = BindableProperty.Create("Height", typeof(GridLength), typeof(RowDefinition), new GridLength(1, GridUnitType.Star),
+		public static readonly BindableProperty HeightProperty = BindableProperty.Create(nameof(Height), typeof(GridLength), typeof(RowDefinition), GridLength.Star,
 			propertyChanged: (bindable, oldValue, newValue) => ((RowDefinition)bindable).OnSizeChanged());
 
 		/// <include file="../../docs/Microsoft.Maui.Controls/RowDefinition.xml" path="//Member[@MemberName='.ctor']/Docs" />
 		public RowDefinition()
 		{
-			MinimumHeight = -1;
+		}
+
+		public RowDefinition(GridLength height)
+		{
+			SetValue(HeightProperty, height);
```

Comment:
(nit)

```suggestion
			Height = height;
```

Reply:
the actual setting of value is done by SetValue, even if you set the property. This is one invocation saved. See this as my contribution to the global warming problem